RoaringBitmap/src/main/java/org/roaringbitmap/IntIterator.java

```diff
@@ -4,28 +4,19 @@
 
 package org.roaringbitmap;
 
+import java.util.PrimitiveIterator.OfInt;
+
 /**
  * A simple iterator over integer values.
  * Using an IntIterator instead of Java's Iterator&lt;Integer&gt;
  * avoids the overhead of the Interger class: on some tests,
  * IntIterator is nearly twice as fast as Iterator&lt;Integer&gt;.
  */
-public interface IntIterator extends Cloneable {
+public interface IntIterator extends Cloneable, OfInt {
   /**
    * Creates a copy of the iterator.
    * 
    * @return a clone of the current iterator
    */
   IntIterator clone();
-
-  /**
-   * @return whether there is another value
-   */
-  boolean hasNext();
```

Comment:
What about keeping these methods, adding .nextInt with a default implementation? it would enable 0.X compatibility.

Reply:
Ok. Both interface has a conflict over .next() (int or Integer).  This was discussed in 

https://github.com/RoaringBitmap/RoaringBitmap/blob/master/RoaringBitmap/src/main/java/org/roaringbitmap/ImmutableBitmapDataProvider.java#L301-L307

-> is the need motivating by this PR covered by wrapping `getIntIterator()` in a `RoaringOfInt` as done in 

https://github.com/RoaringBitmap/RoaringBitmap/blob/master/RoaringBitmap/src/main/java/org/roaringbitmap/ImmutableBitmapDataProvider.java#L81-L87
?